library/src/commonMain/kotlin/com/ioki/passenger/api/models/ApiProvider.kt

```diff
@@ -28,8 +28,15 @@ public data class ApiProvider(
     val avatar: ApiAvatar?,
     @SerialName(value = "avatar_darkmode")
     val avatarDarkmode: ApiAvatar?,
-    @SerialName(value = "other_url") val otherUrl: String?,
+    @SerialName(value = "custom_urls") val customUrls: List<CustomUrl>,
 ) {
+    @Serializable
+    public data class CustomUrl(
+        val name: String,
+        val url: String,
+        @SerialName(value = "sort_key") val sortKey: Int,
```

Comment:
Since when do we get a sort key? 🤔 Can't we make sure that the stuff is ordered by default? 🤔